input/pagecontent/general-requirements.md

```diff
@@ -396,9 +246,11 @@ AU Core Procedure|Procedure.reasonCode, Procedure.reasonReference
 
 ##### Must Support - Choice of Terminology
 
-In AU Core elements that define support for more than one terminology only apply to the [Coding](http://hl7.org/fhir/R4/datatypes.html#Coding) part of the element and are not intended to prevent systems from supplying only a text value. In such cases, the AU Core Responders **SHALL** be able to populate from at least one supported terminology, and AU Core Requesters **SHALL** support processing resources with all supported terminologies. 
+In AU Core elements that define support for more than one code system only apply to the [Coding](http://hl7.org/fhir/R4/datatypes.html#Coding) part of the element and are not intended to prevent systems from supplying only a text value. In such cases, the 
+- AU Core Responders **SHALL** correctly populate the element from at least one supported code system
+- AU Core Requesters **SHALL** accept resources without error an element containing any valid code system. 
```

Comment:
Incomplete sentence - suggest changing to something like: "AU Core Requesters SHALL accept resources without error if the element is present containing any valid code system."

Reply:
replaced with "accept resources without error if the element is present and containing any valid value"